### PR TITLE
FOGL-6979 - Allow for 0 rows updated in an update query to be treated as a normal condition

### PR DESCRIPTION
**Pull request by MarkRiddoch**
We need to see if this fixes the issue with FogLAMP Manage deployments

### CODE REVIEW

**Review by pintomax — APPROVED** [review]

File: C/plugins/storage/postgres/connection.cpp [hunk 1]
```diff
@@ -1286,6 +1287,21 @@ SQLBuffer	sql;
 					return false;
 				}
 			}
+			if (iter->HasMember("modifier") && (*iter)["modifier"].IsArray())
+			{
+				const Value& modifier = (*iter)["modifier"];
+				for (Value::ConstValueIterator modifiers = modifier.Begin(); modifiers != modifier.End(); ++modifiers)
+                		{
+					if (iter->IsString())
```
Comment (ashish-jabble) [inline C/plugins/storage/postgres/connection.cpp:1295]: `modifiers->`

File: C/plugins/storage/postgres/connection.cpp [hunk 2]
```diff
@@ -1286,6 +1287,21 @@ SQLBuffer	sql;
...
+					{
+						string mod = iter->GetString();
```
Comment (ashish-jabble) [inline C/plugins/storage/postgres/connection.cpp:1297]: `modifiers->`

**Review by ashwinscale — APPROVED** [review]
LGTM review by test

**Comment by praveen-garg** [comment]
@MarkRiddoch SQLite LB plugin is not required to be fixed here?

**Comment by MarkRiddoch** [comment]
> @MarkRiddoch SQLite LB plugin is not required to be fixed here?

Yes it does. It used to be the case that the common directory was shared, hence common. It appears that somewhere along the line this connection between the two was dropped and now they have individual common directories - which is a bit of a farce. The least risky thing to do at this stage is apply the change to the sqlitelb common directory as well.

**Review by pintomax — APPROVED** [review]

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**